test/nodetool/test_nodetool.py

```diff
@@ -29,3 +31,16 @@ def test_jmx_compatibility_args(nodetool, scylla_only):
              expected_requests=dummy_request)
     nodetool("compact", "system_schema", "--print-port",
              expected_requests=dummy_request)
+
+
+def test_nodetool_no_args(nodetool_path, scylla_only):
+    res = subprocess.run([nodetool_path, "nodetool"], capture_output=True, text=True)
+
+    sys.stdout.write(res.stdout)
+    sys.stderr.write(res.stderr)
```

Comment:
Same here, the printout is (I think) redundant because assert will print it anyway.

Reply:
Will remove.

Reply:
Removed in v2.